### PR TITLE
DBP-1012 setup loadtest env- First Merge to register GHA in repo, WIP !

### PR DESCRIPTION
**Pull request by MWesterholz**
work in progress, merge might contain bugs. Must be merged to test GHA

### CODE REVIEW

File: .github/workflows/Bap-spsh-k6-loadtest.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: Build and push spsh-k6-loadtest Images
```
Comment (aimee-889) [inline .github/workflows/Bap-spsh-k6-loadtest.yml:1]: please delete this

Comment (MWesterholz) [reply .github/workflows/Bap-spsh-k6-loadtest.yml:1]: done

File: .github/workflows/publish-image-and-helm-on-tag.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      image_name: "spsh-k6-loadtest"
+      run_trivy_scan: true
+      image_tag_generation: "version_git_tag"
+      add_latest_tag: true
+      container_registry: "ghcr.io"
+  publish_helm:
+    uses: dBildungsplattform/dbp-github-workflows/.github/workflows/chart-release.yaml@5.0.0
+    secrets: inherit
+    with:
+      chart_name: spsh-k6-loadtest
```
Comment (aimee-889) [inline .github/workflows/publish-image-and-helm-on-tag.yml:29]: chart name is schulportal-load-tests

File: .github/workflows/publish-image-and-helm-on-tag.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+jobs:
+  publish_docker:
+    permissions:
+      packages: write
+      security-events: write
+      contents: read
+    uses: dBildungsplattform/dbp-github-workflows/.github/workflows/image-publish-trivy.yaml@5.0.0
+    with:
+      image_name: "spsh-k6-loadtest"
```
Comment (aimee-889) [inline .github/workflows/publish-image-and-helm-on-tag.yml:20]: i would also like the image name to change to the repo name: schulportal-load-tests

File: charts/schulportal-load-tests/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+name: spsh-load-test
```
Comment (aimee-889) [inline charts/schulportal-load-tests/Chart.yaml:2]: plaese adjust name

File: .github/workflows/install-and-run-spsh-loadtest.yml [hunk 5]
```diff
@@ -0,0 +1,71 @@
+---
+name: Install and Run SPSH loadtest Action 
+
+on:
+  workflow_call:
+    inputs:
+      instance_name:
```
Comment (aimee-889) [inline .github/workflows/install-and-run-spsh-loadtest.yml:7]: inputs need cleaning

**Review by aimee-889 — APPROVED** [review]
adjustments can be done after merging :)

**Comment by MWesterholz** [comment]
naming adjusted according to review

**Pull request merged**